app/Console/Commands/Cloud/CloudFixSubscription.php

```
@@ -0,0 +1,879 @@
+            } catch (\Exception $e) {
+                $this->error("      - Error searching for email {$email}: ".$e->getMessage());
+            }
+        }
+
+        return null;
+    }
+
+    /**
+     * Handle found subscription update (only for active/past_due subscriptions)
+     */
+    private function handleFoundSubscription($team, $subscription, $foundSub, $searchMethod, $isDryRun, $shouldFix, &$stats)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$team'. (undefined)

(UnusedFormalParameter)